src/components/ec-inline-actions/ec-inline-actions.vue

```diff
@@ -43,7 +43,7 @@
                     v-if="item.icon"
                     :name="item.icon"
                     :size="24"
-                    :type="item.iconType ? item.iconType : null"
+                    :type="item.iconType ? item.iconType : undefined"
```

Comment:
```suggestion
                    :type="item.iconType ?? undefined"
```

Reply:
Actually, iconType is already marked as optional and ec-icon's type can accept undefined already, so it should be just `:type="item.iconType`. Fixed!